### PR TITLE
Show meaningful error when 'username' is already taken in Admin

### PR DESCRIPTION
**Pull request by peter-ochieng**
# Description

[description]

medic/medic-webapp#4209

# Review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- [ ] Documented: Announced in Changes.md in plain English. Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

### CODE REVIEW

File: admin/src/js/controllers/edit-user.js [hunk 1]
```diff
@@ -270,35 +298,41 @@ angular.module('controllers').controller('EditUserCtrl',
...
-                  $rootScope.$broadcast('UsersUpdated', $scope.editUserModel.id);
+                  $rootScope.$broadcast(
+                    'UsersUpdated',
+                    $scope.editUserModel.id
+                  );
                   $uibModalInstance.close();
                 })
                 .catch(function(err) {
-                  $scope.setError(err, 'Error updating user');
+                  $scope.setError(err, err.data.error);
```
Comment (garethbowen) [inline admin/src/js/controllers/edit-user.js]: The error message is translated in the modal template but I think the `err.data.error` from API is just a plain text string. This means if you view the admin app in say Swahili the error message will still show in English. To support translation you could add translation keys to the response from API and use that here.

File: webapp/src/templates/partials/contacts_content.html [hunk 2]
```diff
@@ -34,7 +34,7 @@ <h2>{{selected.doc.name}}</h2>
             <div class="col col-xs-12 col-sm-{{field.width || 12}}" ng-repeat="field in selected.summary.fields">
               <div class="cell">
                 <span ng-if="field.icon" class="field-icon" ng-bind-html="field.icon | resourceIcon"></span>
-                <div>
+                <div class="{{field.label}}">
```
Comment (garethbowen) [inline webapp/src/templates/partials/contacts_content.html]: I think these changes are for a different issue?

**Comment by garethbowen** [comment]
@dianabarsan You've been assigned for code review - can you have a look when you get a chance?

**Comment by dianabarsan** [comment]
@garethbowen I think it was an error that I was assigned for code review on this PR, as all the commits  were made in a different PR targeting the same issue. 
@peter-ochieng do you think we should keep both PRs open?

**Comment by peter-ochieng** [comment]
@dianabarsan .No it shouldn't be open,I'll close it

**Pull request closed without merging**